build/js/main.js

```diff
@@ -1177,89 +1199,80 @@ class Bluetooth extends Plugin {
 
     scanForDevices() {
         this.status(`Start scanning`);
-        api.putPlugin(this.callsign, 'Scan', null, (err, resp) => {
-            if (err !== null) {
-                console.error(err);
-                return;
-            }
-
-            // update after 2s
-            setTimeout(this.update.bind(this), 2000);
-
-            // update discovered device list in every 4s
-            this.Timer = setInterval(this.getDiscoveredDevices.bind(this), 4000);
-
-            this.status(`Scanning...`);
-            // stop scan after 1 min
-            setTimeout(this.stopScan.bind(this), 60000);
-
-        });
-    }
-
-    stopScan() {
-        this.status(`Stopping Scan`);
-        clearInterval(this.Timer);
-        api.putPlugin(this.callsign, 'StopScan', null, (err, resp) => {
+        var f = document.getElementById("BT_DeviceType");
+        var device = f.options[f.selectedIndex].value;
+        api.putPlugin(this.callsign, 'Scan/?LowEnergy='+device, null, (err, resp) => {
             if (err !== null) {
                 console.error(err);
                 return;
             }
-
-            setTimeout(this.update.bind(this), 2000);
-            this.status(`Scan stopped`);
+	    document.getElementById('BT_ScanForDevices').disabled = true;
+            document.getElementById('BT_Pair').disabled = false;
+            setTimeout(this.update.bind(this), 100);
+            // update every 3s
+            this.Timer = setInterval(this.update.bind(this), 3000);
```

Comment:
Be sure to clear the interval on unfocus, else this will keep continuing if the user navigates to another tab.